app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -11,24 +22,31 @@ export default class MoreConfig extends Component {
 
   handleSeriesButtonClick = e => {
     e.preventDefault();
-    this.props.onConfigChange(e);
+    const { onConfigChange } = this.props;
+    onConfigChange(e);
   };
 
   render() {
-    const { onExit, passedData, onSaveDraft } = this.props;
+    const {
+      onExit,
+      passedData: { published, allSeries, canonicalUrl, series },
+      onSaveDraft,
+      onConfigChange,
+    } = this.props;
```

Comment:
these should be added to the proptypes at the bottom of the file

Reply:
yeah ..added in revamp changes commit  @rhymes